politeiawww/api/v1/api.md

```diff
@@ -1396,13 +1396,14 @@ Reply:
 
 Retrieve proposal and its details.
 
-**Routes:** `GET /v1/proposals/{token}`
+**Routes:** `GET /v1/proposals/{token}?version={proposalversion}`
```

Comment:
Looking at the documentation of some of the other `GET` requests with query params, it doesn't like they are including the query params when specifying the route.  See `Unvetted` or `Vetted` as an example.

```suggestion
**Routes:** `GET /v1/proposals/{token}`
```

Reply:
Yes, you're right, they aren't included in the other examples. Wouldn't be better to have it mentioned though? For me, looks pretty handy to have the information about the available query parameters.

Reply:
I would argue that its redundant since the query params are listed in the Params table right below, but I don't think its a big deal either way.  Your call.

Reply:
Alright, I agree with you. I update it but I added the version parameter in the example.